books/RayTracingInOneWeekend.html

```diff
@@ -1864,34 +1896,23 @@
 
 </div>
 
-They also might be absorbed rather than reflected. The darker the surface, the more likely
-absorption is. (That’s why it is dark!) Really any algorithm that randomizes direction will produce
-surfaces that look matte. One of the simplest ways to do this turns out to be exactly correct for
-ideal diffuse surfaces. (I used to do it as a lazy hack that approximates mathematically ideal
-Lambertian.)
-
-(Reader Vassillen Chizhov proved that the lazy hack is indeed just a lazy hack and is inaccurate.
-The correct representation of ideal Lambertian isn't much more work, and is presented at the end of
-the chapter.)
-
 <div class='together'>
```

Comment:
This is a big chunk to require printing to a single page. I think the prior optional break should be fine.

Reply:
Don't sweat it. I have a release work item to review all of these.

Reply:
I'm going to close this out and have you just go through and fix all of this when you go through everything